tasks/configureApache.yml

```diff
@@ -4,37 +4,83 @@
 - name: prepare all required variables
   set_fact:
     httpd_conf_file: "{{ apache_dir }}/{{ ( ansible_os_family == 'Debian' ) | ternary( 'apache2','conf/httpd' ) }}.conf"
-    ssl_conf_file: '{{ apache_dir }}/conf.d/ssl.conf'
+    ssl_conf_file: "{{ apache_dir }}/{{ ( ansible_os_family == 'Debian' ) | ternary( 'conf-available','conf.d' ) }}/ssl.conf"
```

Comment:
Why do we need to set these variables during play? Why don't simple include_vars with corresponding variables?

Reply:
Fixed